ui/prometheus-plugin/src/utils/utils.test.ts

```diff
@@ -107,3 +113,44 @@ describe('formatSeriesName', () => {
     expect(formatSeriesName(inputFormat, metric)).toEqual(output);
   });
 });
+
+describe('getUniqueKeyForPrometheusResult', () => {
+  let labels: { [key: string]: string } = {};
+  beforeEach(() => {
+    labels = {};
+  });
+
+  it('should be a formatted prometheus string', () => {
+    labels = { ['foo']: 'bar' };
+    const result = getUniqueKeyForPrometheusResult(labels);
+    expect(result).toEqual('{foo="bar"}');
+  });
+
+  it('should be formatted with "', () => {
+    labels = { ['foo']: 'bar' };
+    const result = getUniqueKeyForPrometheusResult(labels, true);
```

Comment:
Seeing it like this kind of makes me want named arguments, like `getUniqueKeyForPrometheusResult(labels, { removeExprWrap: true })`

Reply:
Yeah I kinda like this better too. Not sure if we need more options, but it also opens the door to more in the future. No strong opinion though